app/batid/services/imports/import_bdnb_2023_01.py

```diff
@@ -0,0 +1,134 @@
+import csv
+import json
+import os
+from pprint import pprint
+
+import psycopg2
+from django.contrib.gis.geos import GEOSGeometry
+from datetime import datetime, timezone
+
+from django.db import transaction, connection
+from psycopg2.extras import execute_values
+
+from batid.models import Candidate, Address
+from batid.services.imports import building_import_history
+from batid.services.source import Source, BufferToCopy
+
+
+def import_bdnd_2023_01_bdgs(dpt):
+    print("## Import BDNB 2023_01 buildings")
+
+    source_id = "bdnb_2023_01"
+
+    building_import = building_import_history.insert_building_import(
+        source_id, None, dpt
+    )
+
+    src = Source(source_id)
+    src.set_param("dpt", dpt)
+    file_path = src.find(f"{dpt}_bdgs.csv")
+
+    with open(file_path, "r") as f:
+        print("- list buildings")
+        reader = csv.DictReader(f)
+
+        candidates = []
+        for row in list(reader):
+            geom = GEOSGeometry(row["geom_batiment_construction"])
+            geom.srid = 4326
```

Comment:
A mon avis, si tu changes juste le SRID sans appeler de "transform", tu ne modifies pas les coordonnées, tu changes juste le SRID qui est renseigné dans l'objet.

Si tu veux rajouter un test sur cet aspect, il faudrait dans les tests faire un assert que la géometrie intersecte un point qu'on donnerait en wgs84 et qui serait choisi au milieu du bâtiment concerné.

```suggestion
            geom = GEOSGeometry(row["geom_batiment_construction"], srid=2154).transform(4326, clone=True)
```

Reply:
Les données que l'on récupère du csv sont déjà en WGS84. Le problème est que le csv ne spécifie par le srid de la géometrie. Je l'indique donc pour que postgis sache quoi faire de cette géométrie. Pas besoin de transformer.